static/app/bootstrap/initializeSdk.tsx

```diff
@@ -169,6 +170,10 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
     },
   });
 
+  if (sentryConfig.environment === 'development') {
+    /* @__PURE__ */ Spotlight.init();
```

Comment:
I don't know if the spaces (autoformatting) destroys the tree shaking here.
We don't want Spotlight in the prod code.

Reply:
Isn't this like the following?

```suggestion
    /*#__PURE__*/ Spotlight.init();
```

also marking the line is fine, webpack is smart enough to recognize it.

Reply:
lets see